src/runtime/router.js

```diff
@@ -105,6 +126,11 @@ export const init = async () => {
 		const body = toVdom(document.body);
 		hydrate(body, rootFragment);
 
+		// Cache the scripts. Has to be called before fetching the head.
+		[].map.call(document.head.querySelectorAll('script'), (script) => {
```

Comment:
```suggestion
		[].map.call(document.head.querySelectorAll('script[src]'), (script) => {
```

Like the other one, right? 🙂

Reply:
Yes, you're right!